testplan/web_ui/testing/src/Common/utils.js

```diff
@@ -447,3 +447,17 @@ export const truncateString = (
   
   return `${startPortion}...${endPortion}`;
 };
+
+/**
+ * Extracts the "workspace" part of the path
+ * @param {String} path
+ */
+export const getWorkspacePath = (path) => {
+  const srcIndex = path.indexOf('/src/');
```

Comment:
This only works in linux

Reply:
🤔

Reply:
(Discussed this offline.) This is a request from management, the full path will be visible in the popup.